aten/src/ATen/native/EmbeddingBag.cpp

```diff
@@ -74,26 +75,46 @@ void index_select_add<float>(const Tensor &select_indices,
                              const Tensor &add_indices,
                              const Tensor &src,
                              Tensor &output,
-                             const Tensor& offsets) {
+                             const Tensor& offsets,
+                             bool include_last_offset) {
   int64_t ddim = src.size(1);
   auto src_data = src.data_ptr<float>();
   auto select_indices_data = select_indices.data_ptr<int64_t>();
   auto output_data = output.data_ptr<float>();
 
   if (isFastPathIndexSelect(src, output)) {
-    caffe2::EmbeddingLookupIdx(
-      /*block_size=*/ddim,
-      /*output_size=*/offsets.numel(),
-      /*index_size=*/select_indices.numel(),
-      /*data_size=*/src.size(0),
-      /*input=*/src_data,
-      /*indices=*/select_indices_data,
-      /*offsets=*/offsets.data_ptr<int64_t>(),
-      /*weights=*/nullptr,
-      /*scale_bias=*/nullptr,
-      /*normalize_by_lengths=*/false,
-      /*out=*/output_data
-    );
+    int64_t output_size = offsets.numel() - 1;
+    auto offsets_data = offsets.data_ptr<int64_t>();
```

Comment:
If this is a pointer, can we explicitly use auto* ?

Reply:
Addressing this in https://github.com/pytorch/pytorch/pull/32548